testsuite/integration/web/src/test/java/org/jboss/as/test/integration/web/security/WebSimpleRoleMappingSecurityDomainSetup.java

```diff
@@ -1,5 +1,18 @@
 package org.jboss.as.test.integration.web.security;
 
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
```

Comment:
Imports re-arranged - the statics should be first not last.  Also if this class is being revisited it also appears to be missing a copyright header.